client/packages/invoices/src/InboundShipment/DetailView/modals/InboundServiceLineEdit/InboundServiceLineEdit.tsx

```diff
@@ -77,24 +84,7 @@ const InboundServiceLineEditComponent = ({
               Icon={<PlusCircleIcon />}
             />
           </Box>
-          <TableProvider
-            createStore={createTableStore}
-            queryParamsStore={createQueryParamsStore({
-              initialSortBy: { key: 'serviceItemName' },
-            })}
-          >
-            <DataTable
-              id="inbound-detail-service-line"
-              columns={columns}
-              data={lines.filter(({ isDeleted }) => !isDeleted)}
-              dense
-              noDataMessage={
-                !serviceItem
-                  ? t('error.no-service-charges')
-                  : t('error.no-results')
-              }
-            />
-          </TableProvider>
+          <MaterialTable table={table} />
```

Comment:
In the original, when there's no data, it just renders some text rather than an empty table:
<img width="921" height="526" alt="Image" src="https://github.com/user-attachments/assets/e927c4f4-63c2-4fcb-9c0c-a0644e26be8d" />

I think this is better in this case, so can we conditionally render the table and display something like the above when there's nothing?